components/hab/install.sh

```diff
@@ -44,12 +42,7 @@ main() {
   create_workdir
   get_platform
   validate_target
-  if use_packages_chef_io "$version"; then
-    download_packages_chef_io_archive "$version" "$channel" "$target"
-  else
-    get_bintray_version
-    download_bintray_archive
-  fi
+  download_packages_chef_io_archive "$version" "$channel" "$target"
```

Comment:
Since there's no reason to make a distinction anymore, you could just call this function `download_packages` now.